rubocop-mdsol.gemspec

```diff
@@ -13,17 +15,14 @@ Gem::Specification.new do |spec|
     "changelog_uri" => "https://github.com/mdsol/rubocop-mdsol/blob/develop/CHANGELOG.md"
   }
 
-  spec.files    = Dir[
-    "CHANGELOG.md",
-    "MIT-LICENSE",
-    "README.md",
-    "rubocop-mdsol.gemspec",
-    "rubocop-rails.yml",
-    "rubocop-rspec.yml",
-    "rubocop.yml"
-  ]
+  spec.files = `git ls-files -z`.split("\x0").reject do |f|
```

Comment:
can you list the files this would include as of this PR? I think we should _not_ ship the `.rubocop.yml` and other dev-specific files for instance.

Reply:
Updated it to remove dot/hidden files 7ee1ab6

<details>
 <summary>Files that will be included</summary>

 ```ruby
["CHANGELOG.md",
 "Gemfile",
 "MIT-LICENSE",
 "README.md",
 "Rakefile",
 "config/default.yml",
 "lib/rubocop-mdsol.rb",
 "lib/rubocop/cop/mdsol/log_with_data.rb",
 "lib/rubocop/cop/mdsol/unless_not_equal.rb",
 "lib/rubocop/cop/mdsol_cops.rb",
 "lib/rubocop/mdsol.rb",
 "lib/rubocop/mdsol/inject.rb",
 "lib/rubocop/mdsol/version.rb",
 "rubocop-mdsol.gemspec",
 "rubocop-rails.yml",
 "rubocop-rspec.yml",
 "rubocop.yml"]
```
</details>